### DISCUSSION TITLE
按教程设置后无法自动推图？

### DISCUSSION THREAD

**Discussion by LunaticDuel** [LmeSzinc/AzurLaneAutoScript#1186] [Q&A] [answered]
运行到推图环节后提示 No active auto search setting found，请问这个怎么解决啊？![QQ截图20220605153654](https://user-images.githubusercontent.com/106904049/172040546-89dc7991-49a2-4ead-947d-5bb4a5fa8864.png)

**Comment by LmeSzinc** [comment, marked as answer]
帐号等级太低了，达到指挥官等级40级，再开启自律寻敌吧

**Reply by LunaticDuel** [reply]
好的，已经在刷级了
